src/pages/app-development/starter-kit/customers.md

```diff
@@ -10,4 +10,39 @@ keywords:
  - Tools
 ---
 
-# Customers data flow
+# `customer` data flow
+
+This page describes the path that data takes as it travels between Adobe Commerce and your backoffice system when using the Adobe Commerce Extensibility Starter Kit.
+
+## Create
+
+Adobe Commerce origination
```

Comment:
Global comment: Can we make this a full sentence?

Adobe Commerce supports the following ways to create a customer.

(I'm not happy wiith that suggestion, but I'm trying to start the sentence with AC and not use passive voice